server/src/test/java/org/kaazing/gateway/server/context/resolve/DefaultClusterContextTest.java

```diff
@@ -467,7 +444,7 @@ private boolean validateSharedBalancer(CollectionsFactory factory, Set<String> b
         Map<String, Set<String>> sharedBalancerMap = factory.getMap(BALANCER_MAP_NAME);
```

Comment:
Should use constant defined in org.kaazing.gateway.server.context.resolve.DefaultServiceContext